src/main/java/com/targomo/client/Constants.java

```diff
@@ -2,7 +2,7 @@
 
 public class Constants {
 
-    public static final int NO_VALUE_ENTRY_TROVE_MAPS = -1;
+    public static final int NO_VALUE_ENTRY_TROVE_MAPS = Integer.MIN_VALUE;
```

Comment:
Can we change this so easily?

Reply:
we have to change it since negative values (like -1) are possible now. If anyone else but us has used it he hopefully is also using this constant